samples/subsys/instrumentation/README.rst

```diff
@@ -0,0 +1,101 @@
+.. zephyr:code-sample:: instrumentation
+   :name: Instrumentation
+
+   Demonstrate the instrumentation subsystem tracing and profiling features.
+
+Overview
+********
+
+This sample shows the instrumentation subsystem tracing and profiling
+features. It basically consists of two threads in a ping-pong mode, taking
+turns to execute loops that spend some CPU cycles.
+
+Requirements
+************
```

Comment:
there seems to be more requirements regarding the host OS of the user.
Or perhaps there should be a `Limitations` section.

Reply:
but that sample can be built and flashed on windows or MacOS, so there are no OS dependency / requirement.
On Windows I can use putty to test the sample, on Mac miniterm.

The Instrumentation subsystem relies on zaru, which doesn't run on windows afaict, and probably not even on Mac.